cirq/google/calibration/phased_fsim.py

```diff
@@ -220,6 +220,22 @@ class PhasedFSimCalibrationRequest(abc.ABC):
     pairs: Tuple[Tuple[Qid, Qid], ...]
     gate: Gate  # Any gate which can be described by cirq.PhasedFSim
 
+    @staticmethod
```

Comment:
why not have this as a private free function?

Reply:
+1

Reply:
Done.